UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/RadioNavigationalWarnings/ShowRadioNavigationalWarnings.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@model IEnumerable<UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.RadioNavigationalWarningsData>
+
+@{
+    Layout = "~/Views/Shared/_MSILayout.cshtml";
+}
+
+@{
+    ViewData["Header"] = "Radio Navigation Warnings";
+}
+
+<div id="rnwInfo" class="alert alert-primary p-4" role="alert">
+    <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">Important information</div>
+    <p>The following NAVAREA 1 and UK Coastal Warnings (WZs) were in force at: 141009 UTC April 22</p>
+</div>
+
+<!--tabs-->
+<ul class="nav nav-tabs msi-tabs" id="rnwTab" role="tablist">
+    <li class="nav-item" role="presentation">
+        <button class="nav-link active" id="allwarnings-tab" data-bs-toggle="tab" data-bs-target="#allwarnings" type="button" role="tab" aria-controls="allwarnings" aria-selected="true">All warnings</button>
+    </li>
+    <li class="nav-item" role="presentation">
+        <button class="nav-link" id="NAVAREA1-tab" data-bs-toggle="tab" data-bs-target="#NAVAREA1" type="button" role="tab" aria-controls="NAVAREA1" aria-selected="false" tabindex="-1">NAVAREA 1</button>
+    </li>
+    <li class="nav-item" role="presentation">
+        <button class="nav-link" id="ukcostal-tab" data-bs-toggle="tab" data-bs-target="#ukcostal" type="button" role="tab" aria-controls="ukcostal" aria-selected="false" tabindex="-1">UK Costal</button>
+    </li>
+</ul>
+<div class="tab-content" id="rnwTabContent">
+    <div class="tab-pane fade show active" id="allwarnings" role="tabpanel" aria-labelledby="allwarnings-tab">
+        <div class="table-responsive-md">
+            <table class="table table-scroll small-first-col rnw-allwarnings-table">
+                <caption class='hidden-caption'>Radio Navigation Warnings All warnings</caption>
```

Comment:
Should be `"`
```suggestion
                <caption class="hidden-caption">Radio Navigation Warnings All warnings</caption>
```

Reply:
Thanks for highlining. done required changes